app/assets/stylesheets/sul-variables.scss

```diff
@@ -0,0 +1,84 @@
+// Colors
+$beige-5-percent: #fbfbf9;
+$beige: #9d9573;
+$black: #000000;
+$blackish: #2e2d29;
+$blue: #00548f;
+$bright-blue: #006CB8;
+$bright-red: #B1040E;
+$brown: #5e3032;
+$cardinal-red: #8c1515;
+$chocolate: #2F2424;
+$clay: #5f574f;
+$cloud: #dad7cb;
+$cool-grey: #4d4f53;
+$dark-red: #820000;
+$driftwood: #b6b1a9;
+$fog: #F4F4F4;
+$gold: #b26f16;
+$gray-80-percent: #ccc;
+$lagunita: #007C92;
+$light-grey: #f3f3f3;
+$pantone-3145: #007C92;
+$legacy-toolbar: #797268;
+$light-sage: #c7d1c5;
+$light-sandstone: #F9F6EF;
+$mint: #009b76;
+$pantone-334: #009B76;
+$mostly-pure-orange: #ff8000;
+$palo-alto: #175e54;
+$poppy: #e98300;
+$pantone-144: #E98300;
+$public-note-yellow: #faf8d2;
+$purple: #53284f;
+$redwood: #8d3c1e;
+$sandhill: #b3995d;
+$sandstone: #d2c295;
+$sky: #0098db;
+$stone: #544948;
+$stone-legacy: #928b81;
+$sun: #eaab00;
+$teal: #00505c;
+$warm-grey: #3f3c30;
+$gray: #3f3c30;
+$white: #ffffff;
+
+//Variables
```

Comment:
Some of these seem very Searchworks-specific. Maybe we are pare it down now, before it gets more confusing about what's actually being used?

Reply:
Sure.. I can clean it up in the AM

Reply:
👍